src/spans/pgSpan.js

```diff
@@ -0,0 +1,28 @@
+import { getBasicChildSpan, PG_SPAN } from './awsSpan';
+
+export const createPgSpan = (spanId, requestMetadata, pgFields) => {
+  const baseSpan = getBasicChildSpan(spanId, PG_SPAN);
+  return {
+    ...baseSpan,
+    started: requestMetadata.started,
+    connectionParameters: {
+      host: pgFields.connectionParameters.host,
```

Comment:
Do these values always exist? `connectionParameters` can't be `undefined`?

Reply:
I think its required, I will re-check this